front/app/api/navbar/types.ts

```diff
@@ -26,6 +26,7 @@ export interface INavbarItem {
   attributes: {
     title_multiloc: Multiloc;
     code: TNavbarItemCode;
```

Comment:
So before, we were fetching ALL PROJECTS and ALL CUSTOM PAGES just so that we could link the navbar items with the appropriate slug. The more sensible solution now is that we just include them in the serializer, saving us from requesting like 250 projects for big platforms 😤